### PR TITLE
482 predict error with batch size

### LINKED ISSUE #482: Predicting error when using deep_predict when using bacth_size != 1
## Subject of the issue

When using `deepreg_prediction` command with --batch_size !=1, the error will occure probably because the number of samples in the test set can not be divisible by the number of batch_size. 

## Your environment

- DeepReg version 49c38a9b9fcfc344a4ef9bc5b5f4a7cc058b1545
(Which is quite old... I'm not sure if this bug still exists in later versions. )

### PR DESCRIPTION
**Pull request by mathpluscode**
# Description

Fixes #482

Tested with

```bash
deepreg_train --gpu "" --config_path config/unpaired_labeled_ddf.yaml --log_dir test
deepreg_predict --gpu "" --ckpt_path logs/test/save/weights-epoch2.ckpt --mode test -b 5
```

## Type of change

What types of changes does your code introduce to DeepReg? _Put an `x` in the boxes that
apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (fix or improvement on the documentation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR.
If you're unsure about any of them, don't hesitate to ask. We're here to help! This is
simply a reminder of what we are going to look for before merging your code._

- [x] I have
      [installed pre-commit](https://deepreg.readthedocs.io/en/latest/contributing/setup.html)
      using `pre-commit install` and formatted all changed files. If you are not
      certain, run `pre-commit run --all-files`.
- [x] My commits' message styles matches
      [our requested structure](https://deepreg.readthedocs.io/en/latest/contributing/commit.html),
      e.g. `Issue #<issue number>: detailed message`.
- [x] I have updated the
      [change log file](https://github.com/DeepRegNet/DeepReg/blob/main/CHANGELOG.md)
      regarding my changes.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mathpluscode** [comment]
@QianyeYang has a look at the modification, it's super simple and it should solve the issue in your branch ^^. And please be the reviewer of this PR.

@YipengHu @acasamitjana @RemiDelaunay if you can also have a look ^^

File: .travis.yml [hunk 1]
```diff
@@ -27,5 +27,5 @@ jobs:
     - name: "Command Tests: Unpaired Unlabeled"
       script:
         - deepreg_train -g "" --config_path config/unpaired_labeled_ddf.yaml --log_dir test
-        - deepreg_predict -g "" --ckpt_path logs/test/save/weights-epoch2.ckpt --mode test
+        - deepreg_predict -g "" --ckpt_path logs/test/save/weights-epoch2.ckpt --mode test -b 3
```
Comment (YipengHu) [inline .travis.yml:30]: what is -b? are you giving a fixed batch size? if this is required, this needs to go to docs

**Review by YipengHu — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mathpluscode) [reply .travis.yml:30]: this is not required, by default it's 1, the docs have it already https://deepreg.readthedocs.io/en/latest/docs/cli.html#id3

In this test, the test data only have 2 samples, which means using a batch_size = 3 would test the case of a not-full batch. Another option is to add one more sample data for testing and use batch_size 2.

**Review by YipengHu — APPROVED** [review]

**Review by acasamitjana — APPROVED** [review]
Looks good to me! Thanks

**Review by QianyeYang — APPROVED** [review]
Works well on my own branch. 👍

**Pull request merged**